db/migrate/20240227184516_update_audit_managed_resources_queue_name.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateAuditManagedResourcesQueueName < ActiveRecord::Migration[6.1]
+  class MiqQueue < ActiveRecord::Base; end
+
+  def up
+    MiqQueue.where(:method_name => "audit_managed_resources").update_all(:method_name => "report_audit_details", :task_id => "report_audit_details", :tracking_label => "report_audit_details")
```

Comment:
Minor, but please wrap this in say_with_time

```suggestion
    say_with_time("Renaming queue items") do
      MiqQueue.where(:method_name => "audit_managed_resources").update_all(:method_name => "report_audit_details", :task_id => "report_audit_details", :tracking_label => "report_audit_details")
    end
```